doc/frameworks/laminas.md

```diff
@@ -0,0 +1,369 @@
+# Integrate the Doctrine Extensions in Laminas
+
+This guide will demonstrate how to integrate the Doctrine Extensions library into a Laminas application.
+
+## Index
+
+- [Getting Started](#getting-started)
+- [Registering Extension Listeners](#registering-extension-listeners)
+- [Registering Mapping Configuration](#registering-mapping-configuration)
+- [Registering Filters](#registering-filters)
+- [Configuring Extensions via Event Listeners](#configuring-extensions-via-event-listeners)
+
+## Getting Started
+
+> [!TIP]
+> This guide is written using the Laminas MVC quick start as the foundation.
+
+Assuming you have already [created your Laminas application](https://docs.laminas.dev/laminas-mvc/quick-start/),
+the next step will be to ensure you've installed this library and the Doctrine libraries you will need.
+
+For Doctrine MongoDB ODM users, this Composer command will install all required dependencies:
+
+```shell
+composer require doctrine/doctrine-module doctrine/doctrine-mongo-odm-module doctrine/mongodb-odm gedmo/doctrine-extensions
+```
+
+For Doctrine ORM users, this Composer command will install all required dependencies:
+
+```shell
+composer require doctrine/dbal doctrine/doctrine-module doctrine/doctrine-orm-module doctrine/orm gedmo/doctrine-extensions
+```
+
+## Registering Extension Listeners
+
+At the heart of the Doctrine Extensions library are the listeners which enable each extension. The below example demonstrates
+how to register and enable all listeners provided by this library.
+
+### Extensions Compatible with all Managers
+
+```php
+<?php
+
+use Gedmo\Blameable\BlameableListener;
+use Gedmo\IpTraceable\IpTraceableListener;
+use Gedmo\Loggable\LoggableListener;
+use Gedmo\Mapping\Driver\AttributeReader;
+use Gedmo\Sluggable\SluggableListener;
+use Gedmo\SoftDeleteable\SoftDeleteableListener;
+use Gedmo\Sortable\SortableListener;
+use Gedmo\Timestampable\TimestampableListener;
+use Gedmo\Translatable\TranslatableListener;
+use Gedmo\Tree\TreeListener;
+use Psr\Clock\ClockInterface;
+use Psr\Container\ContainerInterface;
+
+return [
+    'service_manager' => [
+        'invokables' => [
+            'gedmo.mapping.driver.attribute' => AttributeReader::class,
+        ],
+        'factories' => [
+            'gedmo.listener.blameable' => function (ContainerInterface $container, string $requestedName): BlameableListener {
```

Comment:
I didn't work with this integration, so this is more a question than a suggestion.

As these functions are not using `$this`, could they be declared as `static`?

Reply:
…maybe? 😅

I don't use Laminas either, so I just went off the docs and some stuff I saw in the Doctrine modules to see if I got this right enough.  It'd be great if there were a Laminas user that could help verify and improve this, but I'm reasonably sure what's here should at least work.

Reply:
I finally took the time to actually set up a Laminas app and validate the docs page.  What's here works.

Declaring the Closures as static works as well (since they don't use `$this`), but, none of the examples on https://docs.laminas.dev/laminas-servicemanager/v4/configuring-the-service-manager/ document them this way so for the sake of simplicity and to stay aligned with the Laminas docs I'm just going to leave these as-is for now.  Folks can optimize this config by using static closures or implementing factory classes if they so desire, but what's here is enough to get everything installed and functional.